utils/api/definitions/enums.ts

```diff
@@ -78,3 +78,9 @@ export enum EstimationSummaryState {
   UNAVAILABLE = 'UNAVAILABLE', // yellow, can not provide any results, contact Service Canada (conditionally eligible)
   AVAILABLE_INELIGIBLE = 'AVAILABLE_INELIGIBLE', // red, display results (ineligible)
 }
+
+export enum LinkLocation {
+  STANDARD = 'STANDARD', // Questions > NeedHelp, and Results > MoreInfo
+  RESULTS_ONLY = 'RESULTS_ONLY', // Results > MoreInfo only
+  RESULTS_APPLY = 'RESULTS_APPLY', // new section above Results > MoreInfo
+}
```

Comment:
Does this mean the more info link is the union of `STANDARD` and `RESULTS_ONLY`

Reply:
Hmm, I think so? I actually need to add another option, I'll do that now and maybe that'll clear things up?

Reply:
So all this needs now is to have a link that is marked as RESULTS_APPLY so the results page will work. This is the link we use: https://www.canada.ca/en/services/benefits/publicpensions/cpp/old-age-security/apply.html